UnitTests/BraintreePayPalTests/BTPayPalClient_Tests.swift

```diff
@@ -383,6 +536,13 @@ class BTPayPalClient_Tests: XCTestCase {
         XCTAssertNotNil(payPalClient.clientMetadataID)
     }
 
+    func testEditFI_whenGenerateFIURLSuccessful_performsSwitchRequest() {
+        let request = BTPayPalVaultEditRequest(editPayPalVaultID: "test-ID")
+        payPalClient.edit(request) { _, _ in }
+
+        XCTAssertNotNil(payPalClient.webAuthenticationSession)
```

Comment:
Are we able to assert that the risk correlation ID is not nil here?

Reply:
We are not persisting it as PayPalClient property anymore but using a local variable.

Reply:
Shouldn't it exist in here?
```swift
payPalClient.edit(request) { result, error in 
    XCTAssertNotNil(result.riskCorrelationID)
    XCTAssertNil(error)
}
```

Reply:
Wanted to make sure this doesn't get missed

Reply:
Did you see a test I pushed?